wire-ios/Wire-iOS/Sources/UserInterface/GroupDetails/Sections/ParticipantsSectionController.swift

```diff
@@ -216,19 +228,54 @@ final class ParticipantsSectionController: GroupDetailsSectionController {
 
     override func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let configuration = viewModel.rows[indexPath.row]
-        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: configuration.cellType.reuseIdentifier, for: indexPath) as! ParticipantsCellConfigurable & UICollectionViewCell
         let showSeparator = (viewModel.rows.count - 1) != indexPath.row
-        (cell as? SectionListCellType)?.sectionName = viewModel.accessibilityTitle
-        (cell as? SectionListCellType)?.cellIdentifier = "participants.section.participants.cell"
-        cell.configure(with: configuration, conversation: conversation, showSeparator: showSeparator)
+
+        let cell = collectionView.dequeueReusableCell(
+            withReuseIdentifier: configuration.cellType.reuseIdentifier,
+            for: indexPath
+        )
+        if let cell = cell as? SectionListCellType {
+            cell.sectionName = viewModel.accessibilityTitle
+            cell.cellIdentifier = "participants.section.participants.cell"
+        }
+
+        lazy var unexpectedCellHandler: () -> UICollectionViewCell = {
+            let message = "Unexpected collection view cell type: \(String(describing: cell.self))"
+            WireLogger.conversation.error(message)
+            assertionFailure(message)
+            return cell
+        }
```

Comment:
The benefit of lazy vars is the same in functions as well as for member variables: if it's not accessed, it will never consumer execution time or memory. The `unexpectedCellHandler` is only called from the two `guard`'s `else` block.

Reply:
I guess the benefit is so little, that it's not worth talking about.
Changed it to `let`.